test/spec/PalindromSpec.js

```diff
@@ -9,10 +9,10 @@ describe("Palindrom", function () {
   });
 
   describe("backward compatibility", function () {
-    it("should work if PuppetJs constructor is used", function (done) {
+    it("should work if Puppet constructor is used", function (done) {
       var initSpy = jasmine.createSpy();
 
-      this.palindrom = new PuppetJs({remoteUrl: window.location.href, callback: initSpy});
+      this.palindrom = new Puppet({remoteUrl: window.location.href, callback: initSpy});
```

Comment:
I am not fond of changing the constructor from `PuppetJs` to `Puppet`. We are supposed to keep backwards compatibility with this change, so we need a `PuppetJs` constructor.